src/app/src/components/AboutSlide.js

```diff
@@ -110,29 +114,32 @@ export default class AboutSlide extends Component {
                         <FontAwesomeIcon icon={['fa', muteIcon]} />
                     </MuteButton>
                 </VideoContainer>
-                <Flex
-                    width={2 / 5}
-                    padding='2rem'
-                    margin='auto'
-                    flexDirection='column'
-                >
-                    <Heading as='h2' variant='small'>
+                <Box width={2 / 5} pl='3rem'>
+                    <Heading
+                        as='h2'
+                        variant='base'
+                        mb='spacious'
+                        lineHeight='medium'
+                    >
                         {title}
                     </Heading>
                     {job && (
                         <>
                             <Heading as='h3' variant='xSmall'>
                                 Name
                             </Heading>
-                            <Text>{name}</Text>
-                            <Text>{job}</Text>
-                            <Heading as='h3' variant='xSmall'>
+                            <Text mb='0'>{name}</Text>
+                            <Text as='em' lineHeight='medium' opacity={0.5}>
+                                {job}
+                            </Text>
+                            <Heading mt='medium' as='h3' variant='xSmall'>
```

Comment:
I am finding these inline styles hard to read, and distracted from the JSX logic. Can we move them to a style block?

Reply:
Could I get more info about what you're imagining instead? Something how `<VideoContainer>` is handled?

Reply:
Something like this, maybe?
<img width="371" alt="Screen Shot 2019-06-06 at 1 27 42 PM" src="https://user-images.githubusercontent.com/5672295/59053219-e57d2c80-885e-11e9-84d6-49e68d0d6db1.png">

<img width="506" alt="Screen Shot 2019-06-06 at 1 27 10 PM" src="https://user-images.githubusercontent.com/5672295/59053190-d302f300-885e-11e9-98bd-7dfa0aefbb7e.png">

Reply:
Yeah, factor out the elements and CSS to standalone constants like `const VideoContainer` or nested like with `const StyledAbout`, https://github.com/azavea/pwd-fishway/pull/99/files/e0e9d664bd1106c56f9b3813254229404b399447#diff-bc9a3980758325b33f8d28b633238d0eR19

Reply:
The multiple `<Text>` elements with inline styling is the part that trips me up the most

Reply:
Cool, I added new named style blocks. Ready for another look.